README.md

```diff
@@ -1478,76 +1478,86 @@ Response:
 HTTP/1.1 200 OK
```

Comment:
There's a lot of noise in these changes, but this does show the start and end dates in the output.